CITATION.cff

```diff
@@ -0,0 +1,35 @@
+# https://citation-file-format.github.io/
+cff-version: 1.2.0
+message: "If you use this software, please cite it using these metadata."
+title: "Alpha: A lazy-grounding Answer-Set Programming system"
+# Authors are sorted:
+# First people, sorted by `family-names`, `given-names`.
+# Second entities, sorted by `name`.
+authors:
+- family-names: "Langowski"
+  given-names: "Michael"
+  affiliation: "TU Wien"
+  #orcid: TODO
```

Comment:
@madmike200590 Do you have an ORCID?

Reply:
Since it probably can't hurt, I got one  now - added it in [70c33f2](https://github.com/alpha-asp/Alpha/pull/314/commits/70c33f2644a44d45ae00588ad5c48f2382ca498c).